### PR TITLE
Docs improviments (Storm integration)

### PR DESCRIPTION
**Pull request by crisdut**
**Description**

After making tests and fixies the integration between `rgb-node` and `storm-node`, I decided to add information about storm integration works.

**Related Issues**

- https://github.com/LNP-WG/lnp-node/pull/59
- https://github.com/RGB-WG/rgb-node/pull/203
- https://github.com/RGB-WG/rgb-node/pull/202
- https://github.com/Storm-WG/storm-node/pull/2

### CODE REVIEW

File: contrib/demo-rgb.sh [hunk 1]
```diff
@@ -142,7 +142,14 @@ rgb psbt analyze ${PSBT}
 # referencing this txid.
 # We also instruct the daemon to send the consignment to the beneficiary LN node.
 # If done on the same system (a self-payment), the --send argument can be omitted.
-rgb-cli -n testnet transfer finalize --endseal ${TXOB} ${PSBT} ${CONSIGNMENT} --send
+# If done on the remote peer, the --send argument is reqquired.
+#   For sending to the remote peer, we need to check if both peers (self and remote) 
+#   have connected by Bifrost protocol (check this using `lnp-cli peers` method).
+NODE_ID="..." #The node_id of the beneficiary LN node
+STORM_ADDR="..." #The stormd IP connected in your RGB node
+STORM_PORT="..." #The stormd RPC port
```
Comment (dr-orlovsky) [inline contrib/demo-rgb.sh:150]: Are these used anywhere? If yes, I think ppl will struggle to repeat the demo in this way; we first need to add `*-cli` commands which would return the node ids, and than put here stuff like ``*_ID=`*-cli node-id` ``

**Review by dr-orlovsky — COMMENTED** [review]
1. Do we need to get all PRs listed in the description to be merged before merging this one?
2. Pls check my review comment

**Comment by crisdut** [comment]
> 1. Do we need to get all PRs listed in the description to be merged before merging this one?

Yes, I could transfer the consignment file only with all the fixes.

> 2. Pls check my review comment

Sure.

File: contrib/demo-rgb.sh (hunk 1 above)
Comment (crisdut) [reply contrib/demo-rgb.sh:150]: Good point!

I will make the changes.

**Review by dr-orlovsky — APPROVED** [review]

**Pull request merged**